sentry-samples/kmp-app/androidApp/src/main/AndroidManifest.xml

```diff
@@ -8,6 +8,7 @@
         android:theme="@style/AppTheme"
         android:name=".SentryApplication">
         <meta-data android:name="io.sentry.auto-init" android:value="false" />
+        <meta-data android:name="io.sentry.attach-screenshot" android:value="true" />
```

Comment:
I'd rather set that on the options directly instead of the Manifest, just for the sake of simplicity for now.